src/fields/Image.jsx

```diff
@@ -3,15 +3,75 @@ var cleanProps = require("./clean-props");
 var MultiplicityField = require('./MultiplicityField.jsx');
 
 module.exports = React.createClass({
+  getInitialState() {
+    console.log(this.props);
```

Comment:
True story: I contemplated commenting "Log" on each of the 3 `console.log`s in this PR and adding that line to the end of the final comment.